### DISCUSSION TITLE
Create the class diagram for BookMyShow

### DISCUSSION THREAD

**Discussion by kanmaytacker** [kanmaytacker/design-questions#21] [General]
What?
Identify the various entities, their attributes, behaviour and relationships with other entities.

Why?
1. Helps identifying and documenting low level implementation
2. Removes uncertainty from actual implementation

How?
Fill out the worksheet [here](https://github.com/kanmaytacker/design-questions/blob/master/notes/06-design-book-my-show-worksheet.md#class-diagrams).

**Comment by sachinsatawkar** [comment]
Entities 

City
  -> Theater name
  -> address


Theater 
  -> movie name
  -> screen number
  ->  time
  
Movie:
    -> title
    -> genre
    ->  screen number
    -> cast
    -> rating

booking
   -> type
    -> number of seat 
     -> show time
    -> movie name 
    -> pay 
   -> time of booking 

 pay 
  -> cash
  -> wallet
  -> online

**Comment by ShitalWebPro** [comment]
![BookMySHow](https://github.com/kanmaytacker/design-questions/assets/118673811/0ff3345d-4a1b-4612-990e-c494b96c8e4f)

**Comment by Manoj-1205** [comment]
![image](https://github.com/kanmaytacker/design-questions/assets/68186238/2598c4a1-1530-4c5a-aaba-429d441849e7)
